docs/guides/auth/email_password.rst

```diff
@@ -449,6 +449,63 @@ an ``auth_token``.
    };
 
 
+Creating a User object
+----------------------
+
+Some applications may want to create a custom ``User`` type in their default
+module to attach application-specific information. You can tie this to an
+``ext::auth::Identity`` by using the ``auth_token`` in our
+``ext::auth::client_token`` global and inserting your ``User`` object with a
+link to the ``Identity``.
+
+For this example, we'll assume you have a 1-to-1 relationship between ``User``
+objects and ``ext::auth::Identity`` objects, but in your own application, you
+may decide to have a 1-to-many relationship.
```

Comment:
```suggestion
.. note::

    For this example, we'll assume you have a one-to-one relationship between
    ``User`` objects and ``ext::auth::Identity`` objects. In your own
    application, you may instead decide to have a one-to-many relationship.
```

This feels less like a core piece of the content and more like a note about the upcoming example. Made some other minor tweaks too.